pages/search.js

```diff
@@ -83,12 +85,49 @@ const SearchFormContainer = styled(Box)`
 
 const DEFAULT_SEARCH_TYPES = ['COLLECTIVE', 'EVENT', 'ORGANIZATION', 'FUND', 'PROJECT'];
 
+const FilterLabel = styled.label`
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  text-transform: uppercase;
+  padding-bottom: 8px;
+  color: #4d4f51;
+`;
+
+const FilterButton = styled(StyledButton).attrs({
+  buttonSize: 'tiny',
+  buttonStyle: 'standard',
+})`
+  height: 22px;
+  background-color: #f1f2f3;
+  margin-right: 8px;
+  margin-bottom: 8px;
+  font-size: 12px;
+  font-weight: 500;
+  width: 84px;
+  cursor: pointer;
+
+  ${props =>
+    props.$isSelected &&
+    css`
+      &,
+      &:active,
+      &:focus {
+        background-color: ${props => props.theme.colors.primary[100]};
+        color: ${props => props.theme.colors.primary[800]};
+      }
+    `}
+`;
+
+const IGNORED_TAGS = ['community', 'user'];
```

Comment:
I've added the two most occurring tags `community` and `user` to this list so that we ignore showing them for now. I think ideally we need to discuss and remove these from db when we are tacking https://github.com/opencollective/opencollective/issues/4764 since we probably don't need these auto created tags anymore.